src/components/collaborator_list/Collaborator.tsx

```diff
@@ -0,0 +1,8 @@
+export interface Collaborator {
+    uuid: string;
+    name: string;
+    eraCommonsId: string;
```

Comment:
What's the use case for requiring an eraCommonsId for lab staff?  Do they already require one?

Are the lab staff internal DUOS users as well?  Should we bind these entities to user objects so we discover the last eraCommonsId they joined their account with?

Reply:
Yes, it's an already existing field on the old collaborators list component.